dim-web/src/routes/dashboard.rs

```diff
@@ -260,3 +149,77 @@ async fn banner_for_show(
         })).collect::<Vec<_>>(),
     }))
 }
+
+pub async fn dashboard(
```

Comment:
God I hate this function, cant believe i wrote it :D. Maybe we could move some of the DB queries into the dim-database crate.